src/components/SeatContest.tsx

```diff
@@ -130,54 +230,127 @@ class SeatContest extends React.Component<Props, State> {
           </Legend>
           <Choices>
             {contest.candidates.map((candidate, index) => {
-              const isChecked = candidate.id === vote
+              const isChecked = candidate.name === vote
               const handleDisabledClick = () => {
                 if (vote && !isChecked) {
                   this.handleChangeVoteAlert(candidate.name)
                 }
               }
               return (
                 <Choice
-                  key={candidate.id}
-                  htmlFor={candidate.id}
+                  key={candidate.name}
```

Comment:
is the change from `id` to `name` because of write-ins? Given how these things are tallied, I suggest instead that we keep `id` as the unique identifier in every case, and use `[WI]` or some other write-in specific ID for the write-in option.

Reply:
There is no tally. This hasn't been a defined feature. Let's convert to IDs or some better option when we have this feature defined.

Reply:
@beausmith I understand there's not tally here, but I still think this is a bad design approach, because I think identifiers should be different than visible names, especially as names might have multiple expression in different character sets. Let's switch this back to `id` please.

Reply:
I'd like us to discuss this more. I know the feeling you're having, I've gone back and forth, and I took this approach as it dramatically simplifies the logic for us and I can't think of a demonstrable reason not to do it. 

Let's merge this so as not to delay before demo and discuss early next week over video or in person and make a plan.

Reply:
Note: the IDs are still in the election.json.